kentikapi/client.go

```diff
@@ -161,8 +158,28 @@ func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error
 	)
 }
 
-func makeTLSOption(c Config) grpc.DialOption {
-	if c.DisableTLS {
+func makeGRPCHostPort(apiURL string) (grpcHostPort string, tlsEnabled bool, err error) {
+	tlsEnabled = false
+	u, err := url.Parse(apiURL)
+	if err != nil {
+		return apiURL, tlsEnabled, err
+	}
+	grpc := u.Host
+	if u.Scheme == "https" {
+		tlsEnabled = true
+		if u.Port() == "" {
+			grpc += ":443"
```

Comment:
Why the code appends missing port only for https case? I think it should add :80 for http.

Reply:
Done